Configuration/TCA/Overrides/pages.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$storageRepository = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('MASK\\Mask\\Domain\\Repository\\StorageRepository');
+$configuration = $storageRepository->load();
+
+if (!empty($configuration) && array_key_exists('pages', $configuration)) {
```

Comment:
sorry, would you mind to test with !empty($configuration['pages'])? array_key_exists() returns also true for $configuration['pages'] = null